### DISCUSSION TITLE
IKEA Tradfri / LED2005R5 has been replaced with LED2106R3

### DISCUSSION THREAD

**Discussion by daNick73** [Koenkk/zigbee2mqtt#19760] [Devices]
Hi

Here in Germany, IKEA Tadfri LED2005R5 has been replaced with new LED2106R3
(they even share the same manual https://www.ikea.com/de/de/manuals/tradfri-led-leuchtmittel-gu10-345-lm-smart-kabellos-dimmbar-weissspektrum__AA-2375179-1-2.pdf)

LED2106R3 model  announces itself as "TRADFRI_bulb_GU10_WS_345lm"
<img width="223" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/21158022/f54ff410-ab0d-43ec-a634-22bdaea7fad7">

Is there a way to duplicate

https://www.zigbee2mqtt.io/devices/LED2005R5.html

For the new devices?

THX!

**Comment by Koenkk** [comment]
Added!

Changes will be available in the dev branch tomorrow (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) and will be included in the next release on 1 December.

**Reply by daNick73** [reply]
simply amazing thx!
